File: WebLabelPrint_VB/Models/Utility.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace WebLabelPrint.Models
   ''' <summary>
   ''' Miscellaneous supporting functions.
   ''' </summary>
   Public Module Utility
      Private _random As New Random()

      ''' <summary>
      ''' Returns a random number between min and max.
      ''' </summary>
      Public Function GetRandom(min As Integer, max As Integer) As Integer
         Return _random.[Next](min, max)
      End Function
   End Module
End Namespace

File: WebLabelPrint_CS/Models/Utility.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebLabelPrint.Models
{
   /// <summary>
   /// Miscellaneous supporting functions.
   /// </summary>
   public static class Utility
   {
      private static Random _random = new Random();

      /// <summary>
      /// Returns a random number between min and max.
      /// </summary>
      public static int GetRandom(int min, int max)
      {
         return _random.Next(min, max);
      }
   }
}